osu.Framework/Graphics/Containers/DelayedLoadUnloadWrapper.cs

```diff
@@ -84,6 +84,14 @@ protected override void EndDelayedLoad(Drawable content)
             });
         }
 
+        private readonly object unloadLock = new object();
+
+        protected override void Dispose(bool isDisposing)
+        {
+            lock (unloadClock)
```

Comment:
```suggestion
            lock (unloadLock)
```

i am very scared by this